contributor-guide/modules/ROOT/pages/requirements/library-metadata.adoc

```diff
@@ -1,197 +1,189 @@
+=== `category`
 
+A list of one or more of the <<Categories>> that the library belongs to.
 
 === `documentation`
 
-
-Path to the documentation, defaults to the root of the
- module.
-
+Path to the documentation, defaults to the root of the module.
 
 === `cxxstd`
 
-
-The minimum pass:[C++] standard compilation level at which
- all, or the large majority, of the functionality in
- the library is usable. The possible values are: 
-
+The minimum pass:[C++] standard compilation level at which all, or the large majority, of the functionality in the library is usable. The possible values are: 
 
 * 98 = pass:[C++]98
-* 03 = pass:[C++]98/pass:[C++]03
+* 03 = pass:[C++]03
 * 11 = pass:[C++]11
 * 14 = pass:[C++]14
 * 17 = pass:[C++]17
 * 20 = pass:[C++]20
-
-
-The level only indicates the minimum level, which
- means that the functionality of the library can be
- used when compiling at that level or at a higher
- level. There may be some functionality in the library
- which will need a higher pass:[C++] standard compilation
- level than is indicated by this value, but the
- information about that specific functionality will be
- provided for the end-user within the documentation for
- that library. If a library does not have this field it
- indicates that the end-user will have to read the
- library documentation to understand what pass:[C++] standard
- compilation level is needed to use the library. 
-
-
-== Available categories
-
-String
-String and text processing
-Containers
-Containers
-Iterators
-Iterators
-Algorithms
-Algorithms
-Function-objects
-Function objects and higher-order programming
-Generic
-Generic Programming
-Metaprogramming
-Template Metaprogramming
-Preprocessor
-Preprocessor Metaprogramming
-Concurrent
-Concurrent Programming
-Math
-Math and numerics
-Correctness
-Correctness and testing
-Error-handling
-Error handling and recovery
-Data
-Data structures
-Domain
-Domain Specific
-Image-processing
-Image processing
-IO
-Input/Output
-Inter-language
-Inter-language support
-Emulation
-Language Features Emulation
-Memory
-Memory
-Parsing
-Parsing
-Patterns
-Patterns and Idioms
-Programming
-Programming Interfaces
-State
-State Machines
-System
-System
-Miscellaneous
-Miscellaneous
-workarounds
-Broken compiler workarounds
+* 23 = pass:[C++]23
+
+The level only indicates the minimum level, which means that the functionality of the library can be used when compiling at that level or at a higher level. There may be some functionality in the library which will need a higher pass:[C++] standard compilation level than is indicated by this value, but the information about that specific functionality will be provided for the end-user within the documentation for that library. If a library does not have this field it indicates that the end-user will have to read the library documentation to understand what pass:[C++] standard compilation level is needed to use the library. 
+
+Note that `03`, `11`, and `14` are commonly set minimum levels.
```

Comment:
This is correct but it might be considered slightly prescriptive for new contributors, which is problematic since boost is no longer supporting C++03 and most libraries are moving to C++11: https://www.boost.org/users/history/version_1_82_0.html

In fact, we could just mention 11 and 14 here since they will already have deprecated C++03 at the time this documentation is online.

Reply:
removed 03